### ISSUE TITLE
mkvirtualenv returns virtualenv already exists wheres not

### ISSUE THREAD

**Issue by saiftheboss7** [davidmarble/virtualenvwrapper-win#85]
but the problem is whenever I run the command mkvirtualenv it throws me an error about the virtualenv already exists, whereas I have not created any virtual environment yet.

![alt text](https://i.stack.imgur.com/XXYsP.png "command mkvirtualenv")

**Comment by Cyntwikip** [comment]
Hi! 

I also encountered this problem earlier. It seems like there's a problem with the new version of virtualenvwrapper-win.
What I did to resolve this issue was I uninstalled the package and then installed the previous version.

Here are the pip commands for doing those:
pip uninstall virtualenvwrapper-win
pip install virtualenvwrapper-win==1.2.0

**Comment by saiftheboss7** [comment]
Thanks @Cyntwikip You saved me from a lot of hassle. I was getting crazy from yesterday night wondering what I am doing wrong. It fixed the issue!

**Comment by thebjorn** [comment]
Looks like the problem is caused by spaces in the `WORKON_HOME` path.

**Comment by thebjorn** [comment]
@saiftheboss7 @Cyntwikip Thank you very much for reporting this issue.  It has now been fixed and will be included in the next release (v1.2.3).  If you want to continue using v1.2.2 until the next version is released, you'll need to point the `WORKON_HOME` environment variable to a directory without spaces.

**Comment by thebjorn** [comment]
v.1.2.3 is now on PyPI.

**Comment by cepbrian** [comment]
I seem to still be having this problem on 1.2.5. 

```
Requirement already up-to-date: virtualenvwrapper-win in c:\users\brianp\appdata\local\programs\python\python37-32\lib\site-packages (1.2.5)
Requirement already satisfied, skipping upgrade: virtualenv in c:\users\brianp\appdata\local\programs\python\python37-32\lib\site-packages (from virtualenvwrapper-win) (16.4.3)

```
The error I'm getting is a little different:

`\venvs\win' was unexpected at this time.`

My WORKON_HOME is C:\Dropbox (CEP)\venvs\win

So it's possible that the problem is the ().

**Comment by vijayusa11** [comment]
> Hi!
> 
> I also encountered this problem earlier. It seems like there's a problem with the new version of virtualenvwrapper-win.
> What I did to resolve this issue was I uninstalled the package and then installed the previous version.
> 
> Here are the pip commands for doing those:
> pip uninstall virtualenvwrapper-win
> pip install virtualenvwrapper-win==1.2.0

----------------------------------
it is not working

**Comment by thebjorn** [comment]
@vijayusa11 "it is not working" is not a problem description.  This is an issue that was closed in 2017. Please open a new issue with all the relevant information to recreate your problem.

**Comment by Cyntwikip** [comment]
@vijayusa11, this is the fix for the bug in version 1.2.2. I agree with @thebjorn that you should open a new issue, and document everything needed to reproduce the error you're encountering.
